pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -188,7 +201,14 @@ func (e *cloudWatchExecutor) checkHealthLogs(ctx context.Context, pluginCtx back
 	if err != nil {
 		return err
 	}
-	_, err = e.handleDescribeLogGroups(ctx, logsClient, simplejson.NewFromAny(map[string]interface{}{"limit": "1"}))
+
+	defaultLimit := int64(1)
```

Comment:
default values should go at the top of the file with the other consts

Reply:
Good to know, thanks!